### PR TITLE
docs(atomic): add header navigation to sample page

### PR DESCRIPTION
**Pull request by liviarett**
https://coveord.atlassian.net/browse/KIT-1571

Screenshot 2022-04-19 at 14 19 35
Screenshot 2022-04-19 at 14 19 51

### CODE REVIEW

File: packages/atomic/src/pages/header.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const getCurrentExample = () =>
+  links.find((link) => link.href === window.location.pathname);
+
+const makeLinks = () => {
+  return links
+    .filter((link) => link.href !== window.location.pathname)
+    .map(
+      (link) =>
+        `<a href="${link.href}" style="margin-right: 10px; color: var(--atomic-primary); text-decoration: none;">${link.label}</a>`
```
Comment (btaillon) [inline packages/atomic/src/pages/header.js:20]: I would wrap the `a` element in a `li` (even though accessibility doesn't matter that much here)

**Review by btaillon — APPROVED** [review]
neat!

File: packages/atomic/src/pages/header.js (hunk 1 above)
Comment (liviarett) [reply packages/atomic/src/pages/header.js:20]: ah yeah, that's what I meant to do! thanks for catching it

**Review by ThibodeauJF — APPROVED** [review]
Hey that's pretty cool! Thank you for that 😄

**Pull request merged**